src/main/java/uk/gov/ons/ctp/response/collection/exercise/util/AggregatedDateFormat.java

```diff
@@ -1,37 +0,0 @@
-package uk.gov.ons.ctp.response.collection.exercise.util;
-
-import lombok.extern.slf4j.Slf4j;
-
-import java.text.DateFormat;
-import java.text.FieldPosition;
-import java.text.ParsePosition;
-import java.util.Arrays;
-import java.util.Date;
-
-@Slf4j
-public class AggregatedDateFormat extends DateFormat {
```

Comment:
Joe mentioned something about moving away from the rm-common-service, if this is the case is there a point in removing this?

Reply:
The common version of this class is used by the CustomObjectMapper which is used by all projects - this is a classic example of something that SHOULD be in a common library (otherwise you'd have to copy and paste all the code into each of the Java services).